src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const thumbs = document.querySelector('#thumbs');
+const largeImg = document.querySelector('#largeImg');
+
+thumbs.addEventListener('click', (e) => {
+  if (e.target.tagName === 'IMG') {
+    e.preventDefault();
```

Comment:
Calling `e.preventDefault()` here assumes that the `IMG` element is inside an anchor (`<a>`) tag. If the `IMG` is not directly inside an anchor, this might not prevent the default action as expected. Ensure that the HTML structure matches this assumption.